### ISSUE TITLE
[Jscpd 3.5.0] @jscpd/html-reporter missing dependency @jscpd/finder

### ISSUE THREAD

**Issue by harrytran998** [kucherenko/jscpd#548]
**Describe the bug**

```js
Error: @jscpd/html-reporter tried to access @jscpd/finder, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

**To Reproduce**
1. Create some project
2. Install jscpd
3. run `yarn jscpd --pattern "src/**/*.ts"`

**Expected behavior**
Everything work well

**Screenshots**
<img width="1271" alt="image" src="https://user-images.githubusercontent.com/24296018/197371260-b0f427eb-8278-45e4-a964-3c1c09e29496.png">

**Desktop (please complete the following information):**
 - OS: MacOs Montery
 - NodeJS Version: 18
 - jscpd version 3.5.0

**Additional context**

**Comment by harrytran998** [comment]
I tried to fork and make a pull request to fix this. Just simple put `"@jscpd/finder": "^3.5.0"` in your dependencies of `@jscpd/html-report` and re-run `yarn install`.

But seem your project configs something I can not install dependencies 😆.

**Comment by kucherenko** [comment]
fixed at jscpd@3.5.8
